src/library/clSPARSEConfig.cmake.in

```diff
@@ -1,3 +1,19 @@
+/* ************************************************************************
```

Comment:
Wrong comment char. There should be "#" instead of "*", as it is a cmake file. I will push a commit fixing that.